src/Controller/Api/PersonController.php

```diff
@@ -462,4 +462,38 @@ public function deleteAction(int $id)
         }
         return $this->makeNoContentResponse();
     }
+
+
+    /**
+     * Returns a single person.
+     *
+     * @param Person $person The id of the Person.
+     *
+     * @Route(
+     *     "/api/person/{id}",
+     *     name="pelagos_api_get_person",
+     *     methods={"GET"},
+     *     defaults={"_format"="json"},
+     *     requirements={"id"="\d+"}
```

Comment:
maybe we could make id optional, and if logged in and left null, it returns the json of the logged-in user..idk. Works well as-is too.